bentoml/_internal/frameworks/xgboost.py

```diff
@@ -78,9 +79,9 @@ def _get_model_info(
 @inject
 def load(
     tag: t.Union[str, Tag],
-    booster_params: t.Optional[t.Dict[str, t.Union[str, int]]] = None,
+    booster_params: t.Dict[str, t.Union[str, int]] = FrozenDict(),
```

Comment:
This solution actually breaks the consensus of typing. We devs see an arg which is Dict, they will assume that they can do `dict.update` on it.

Reply:
but the idea of FrozenDict is that `update` or all ops is forbidden if the given parameters is not given?